src/careamics/file_io/read/get_func.py

```diff
@@ -0,0 +1,56 @@
+"""Module to get read functions."""
+
+from pathlib import Path
+from typing import Callable, Dict, Protocol, Union
+
+from numpy.typing import NDArray
+
+from careamics.config.support import SupportedData
+
+from .tiff import read_tiff
+
+
+# This is very strict, arguments have to be called img
```

Comment:
What does "img" mean in this context?

Reply:
oh, it's probably `file_path` here!

Reply:
Ah yes I think I copy and pasted my comment from the `WriteFunc` description incorrectly, will update